tsconfig.json

```diff
@@ -2,10 +2,12 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "noImplicitAny": false
```

Comment:
We can't turn off `noImplicitAny` - happy to help through whatever compiler issues it brings up, though honestly I think we should just stick this in a `tools` directory as JavaScript and call it a day

Reply:
Np! I get the following when I remove it:

```
$ npm run compile

> @google-cloud/storage@3.0.4 compile /Users/deklerk/workspace/nodejs-storage
> tsc -p . && cp src/bin/storage.proto build/src/bin/storage.proto

src/bin/benchwrapper.ts:44:15 - error TS7006: Parameter 'call' implicitly has an 'any' type.

44 function read(call, callback) {
                 ~~~~

src/bin/benchwrapper.ts:44:21 - error TS7006: Parameter 'callback' implicitly has an 'any' type.

44 function read(call, callback) {
                       ~~~~~~~~

src/bin/benchwrapper.ts:60:16 - error TS7006: Parameter 'call' implicitly has an 'any' type.

60 function write(call, callback) {
                  ~~~~

src/bin/benchwrapper.ts:60:22 - error TS7006: Parameter 'callback' implicitly has an 'any' type.

60 function write(call, callback) {
                        ~~~~~~~~

src/bin/benchwrapper.ts:67:19 - error TS7053: Element implicitly has an 'any' type because expression of type '"StorageBenchWrapper"' can't be used to index type 'GrpcObject | typeof Client | Message<object>'.
  Property 'StorageBenchWrapper' does not exist on type 'GrpcObject | typeof Client | Message<object>'.

67 server.addService(storageBenchWrapper['StorageBenchWrapper']['service'], {
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 10 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/storage@3.0.4 compile: `tsc -p . && cp src/bin/storage.proto build/src/bin/storage.proto`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @google-cloud/storage@3.0.4 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/deklerk/.npm/_logs/2019-07-31T22_25_00_451Z-debug.log
```

Reply:
Decided to not use TypeScript for this, so not a concern :) Should be able to just remove this

Reply:
Done